### PR TITLE
Qr feature

### LINKED ISSUE #10: QR code for shortened URL
Return a QR code of a shortened URL

### PR DESCRIPTION
**Pull request by luisgg98**
Now creating a qr code is available on the website and it has been tested using Mocks

### CODE REVIEW

**Comment by jgarciapueyo** [comment]
I have revised the Pull Request. However, because I do not have experience with tests (unit test of controllers and services, or integration tests), I have asked @AlbertoCalvoRubio to finish revising the tests of your PR.

File: src/main/java/urlshortener/web/QRController.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import urlshortener.service.SafeBrowsingService;
+
+@Controller
+public class QRController {
+
+  private final SafeBrowsingService safeBrowsingService;
+
+  public QRController(SafeBrowsingService safeBrowsingService) {
+
+    this.safeBrowsingService = safeBrowsingService;
```
Comment (AlbertoCalvoRubio) [inline src/main/java/urlshortener/web/QRController.java]: I think you don't use this service here

**Review by AlbertoCalvoRubio — CHANGES_REQUESTED** [review]
- I have written a comment in the code. Please change that
- You should write some test about Service Testing at .. test/java/urlshortener/service , it may take a few lines
- We should wait for Pelicer's feedback about the bug pointed out by SonarCloud

**Comment by luisgg98** [comment]
I've corrected what I was doing wrong and removing the unnecessary code or files.
I think it's finally done. Thank you guys for your time and patience.

**Comment by luisgg98** [comment]
I've corrected what I was doing wrong and removed the unnecessary code or files.
I think it's finally done. Thank you guys for your time and patience.

**Review by AlbertoCalvoRubio — APPROVED** [review]

**Comment by AlbertoCalvoRubio** [comment]
Luis try to fix the vulnerability indicated by SonarCloud

**Comment by luisgg98** [comment]
According to what the teacher told us I have to make a lots of changes on the code, so I'll start a new PR soon

**Pull request closed without merging**